CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ option(BUILD_SHARED_LIBS "Build shared avif library" ON)
 option(AVIF_ENABLE_WERROR "Treat all compiler warnings as errors" ON)
 
 option(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R "Enable experimental YCgCo-R matrix code" OFF)
+option(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP "Enable experimental gain map code (for HDR images that look good both on HDR and SDR displays)" OFF)
```

Comment:
Nit: Please add a comment somewhere (either in this file or in avif.h) to specify the ISO proposal/contribution that specifies the gain maps implemented in this pull request.

Reply:
Done in avif.h
I don't think a link exists but I added the name of the (draft) spec.